src/assets/Shares.sol

```diff
@@ -35,6 +35,14 @@ contract Shares is Asset, SharesInterface {
     // PUBLIC VIEW METHODS
 
     function getName() view returns (string) { return name; }
+
+    // Function to convert from string to bytes32
+    function getNameinBytes32() view returns (bytes32 result) {
+        bytes memory localName = bytes(name);
+        assembly {
+          result := mload(add(localName, 32))
+        }
+    }
```

Comment:
is this needed?

Reply:
Yes, I used that to convert from string to bytes32 so that I can get the name in bytes32. In funds.sol the name is returned in the form of bytes32. So i had to change that in shares.sol.

Reply:
why did u need the name in bytes32?

Reply:
Because the string can't be used in funds.sol, I had to use bytes32.

Reply:
Think u meant FundRanking.sol

Reply:
Sorry, yes!